docs/src/main/asciidoc/_cloud/References.adoc

```diff
@@ -0,0 +1,23 @@
+# References
+
+# Helm
+* https://helm.sh/[Official page]
+* https://helm.sh/docs/intro/install/[Installing Helm]
+* https://github.com/wildfly/wildfly-charts/blob/main/charts/wildfly/README.md[Helm Chart for WildFly]
+** https://www.wildfly.org/news/2021/05/05/helm-charts-for-wildfly/[Related blog post]
+* https://docs.wildfly.org/wildfly-charts/[Helm Charts for WildFly]
+
+# OpenShift
+* https://access.redhat.com/downloads/content/290/ver=4.12/rhel---9/4.12/x86_64/product-software[Download OpenShift]
+* https://developers.redhat.com/products/openshift-local/overview[Red Hat OpenShift Local]
+* https://www.redhat.com/en/technologies/cloud-computing/openshift/try-it[Try Red Hat OpenShift]
+* https://docs.okd.io/4.11/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands[Getting started with the OpenShift CLI]
```

Comment:
This points to OKD 4.11. Could we sync the link between the OCP doc and the `oc` doc?

Reply:
It would be nice but I'm not sure how to achieve this

Reply:
Done